Game of Game of Life.py

```diff
@@ -81,7 +70,9 @@ def CheckUserInput():
 Screen=pg.display.set_mode((Size*Width,Size*Height))
 Screen.fill(Background)
 pg.display.set_caption("Game of Life")
-Board=CleanBoard()
+Board=CleanBoard()#A 2d matrix representing the board
```

Comment:
Not really even a mistake, but there is no need for extra comments on self-explanatory stuff. If I see `Board=CleanBoard()` I know that this is the game board and it is empty. If I want to know the variable type of the board I go and look at the function. Read this https://www.python.org/dev/peps/pep-0008/#comments

Reply:
done